manifest-celery.yml.j2

```diff
@@ -0,0 +1,29 @@
+---
+applications:
+- name: notify-template-preview-celery
+
+  disk_quota: 2G
+  memory: 2G
+  health-check-type: process
+  command: exec scripts/run_app_paas.sh celery -A run_celery.notify_celery worker --loglevel=INFO --concurrency=5 --uid=`id -u celeryuser`
+
+  services:
+    - logit-ssl-syslog-drain
+
+  routes:
+    - route: {{ environment }}-notify-template-preview-celery.cloudapps.digital
+
+
+  env:
+    NOTIFY_ENVIRONMENT: {{ environment }}
+
+    NOTIFY_APP_NAME: template-preview-celery
```

Comment:
Good question - I had a look at this and notifications-api has different values of `CW_APP_NAME` for each of its apps, so we do different things in different repos. Going to keep this as different here so that we get different log groups per app.